src/staticanalysis/bot/static_analysis_bot/config.py

```diff
@@ -65,8 +65,10 @@ def setup(self,
               cov_config=None):
         self.app_channel = app_channel
         self.download({
-            'cpp_extensions': frozenset(['.c', '.h', '.cpp', '.cc', '.cxx', '.hh', '.hpp', '.hxx', '.m', '.mm']),
+            'cpp_extensions': frozenset(['.c', '.h', '.cpp', '.cc', '.cxx', '.m', '.mm']),
```

Comment:
Shouldn't you remove the `.h` from `cpp_extensions` then ?

Reply:
this is what happens when yo do this saturday morning :)